harness/determined/experimental/_native.py

```diff
@@ -97,7 +97,7 @@ def _make_test_workloads(
     check.eq(len(batch_metrics), config.scheduling_unit())
     logging.debug(f"Finished training, metrics: {batch_metrics}")
```

Comment:
Can you also make this logging.debug into an info while you are here?  Also below with the validation metrics.